packages/amplify_auth_cognito/android/src/main/kotlin/com/amazonaws/amplify/amplify_auth_cognito/types/FlutterAuthErrorMessage.kt

```diff
@@ -72,6 +72,11 @@ enum class FlutterAuthFailureMessage {
       return "AMPLIFY_CONFIRM_PASSWORD_FAILED"
     }
   },
+  CURRENT_USER{
```

Comment:
Not important right now, but you can change this to enums similar to how I've done in the storage category. You do not need to override as well.